R/fusion_standardization.R

```diff
@@ -29,15 +32,33 @@
 #'   caller = "STARFUSION",
 #'   tumorID = "tumorID"
 #' )
+#' # read in CUSTOM type file
+#' formattedCUSTOM <- fusion_standardization(fusionfileStarFusion,
+#'   caller = "CUSTOM",
+#'   tumorID = "All",
+#'   input_json_file = system.file("extdata", "config", package = "annoFuseData")
+#' )
```

Comment:
Can you please describe the issue so that I can look into it? Did you had an error pulling the config file from annoFuseData package?

Reply:
@federicomarini Regarding your comment on warnings, I had few print statements in. Can you share the warning  while running the custom type ?

Reply:
About the example: The error is triggered by simply running the code in the @examples section.
When running R CMD check, this is done automatically, so I get something like this:

<img width="689" alt="image" src="https://user-images.githubusercontent.com/7447478/191264587-f58e97a6-d7b4-4f43-b91c-e8d06efd7d8d.png">

Reply:
Aha I see! We have this error because code is unable to find config file in annoFusedata package. We haven't released the latest version of annoFusedata package to CRAN server. I believe once we release annoFusedata package to CRAN, this will be fixed

Reply:
Hmm, not really I guess. I do have the latest version of annoFuseData installed, and it is correctly picked.
Otherwise I would have had a different error message - the one telling me there is no file with the specs in it.

To me, the culprit seems to be that the fusionfileStarFusion does not have the columns required.
Also, from the live run of the lines:

<img width="664" alt="image" src="https://user-images.githubusercontent.com/7447478/191267206-7eb15c7e-50dd-4acd-90c1-b1ea4d760ca3.png">

I can suggest you use `message` instead of `print` for these statements

```
[1] "Annotating based on config file"
[1] "All required columns exists!"
```

Reply:
nvm, the print->message has just been done :)

Reply:
(but the error above stays)

Reply:
@federicomarini  Can you also share input data to this function ?

Reply:
I am running the example from top to bottom, simply that.
Like in "start R, load annoFuse, execute this series of commands line by line"

Reply:
(which is exactly what R CMD check does to do all the checks and see if this works "just like we said")

Reply:
I run it and was unable to reproduce the error 
`output=fusion_standardization(dataframe_custom,"CUSTOM","ALL",input_json_file = system.file("extdata", "config", package = "annoFuseData"))`

Reply:
Well, that is different from what I had to run in the example 😉 
In the current status you would still need to define (and potentially put in the annoFuseData an example for that) the `fusionfileCustom` path. Otherwise, I would (correctly) get an error
<img width="668" alt="image" src="https://user-images.githubusercontent.com/7447478/191294023-977622e9-d979-4b96-add2-a3fd85e05a5a.png">

Actually, you can also create that data.frame on the fly in the example, if it makes sense to do so.
That's pretty much up to you 😃

Reply:
Sure, I'm not debating that. It is only that you need a "real example" of a mini data frame that gets passed in this step. Otherwise the check of the example is going to return an error.

Just do something like 

```
fusionfileCustom <- data.frame(
  col1_you_need = c(,,,,),
  col2_you_need = c(,,,,),
  col3_you_need = c(,,,,)
)
```
... in the lines before the call to `fusion_standardization`. I am putting a placeholder in the code